oracle/metadata.csv

```diff
@@ -6,7 +6,7 @@ oracle.shared_pool_free,gauge,,percent,,shared pool free memory %,0,oracle,share
 oracle.physical_reads,gauge,,reads,sec,physical reads per sec,0,oracle,phys reads
 oracle.physical_writes,gauge,,writes,sec,physical writes per sec,0,oracle,phys reads
 oracle.enqueue_timeouts,gauge,,timeouts,sec,enqueue timeouts per sec,0,oracle,enqueue timeouts
-oracle.gc_cr_receive_time,gauge,,blocks,sec,GC CR receive time,0,oracle,gc cr receive time
+oracle.gc_cr_block_received,gauge,,blocks,sec,GC CR receive time,0,oracle,gc cr receive time
```

Comment:
Should we also change the description and short name?

Reply:
awesome catch, thank you ofek!